README.md

```diff
@@ -1,54 +1,65 @@
 
-MetricFlow is a computational framework for building and maintaining consistent metric logic. The name comes from the approach taken to generate metrics. Using the user-defined semantic model, a query is first compiled into a metric dataflow plan. The plan is then converted to an abstract SQL object model, optimized, and rendered to engine-specific SQL.
+<p align="center">
+<img src="https://github.com/transform-data/metricflow/raw/main/assets/example_plan.svg" height="500"/>
+<br /><br />
+</p>
 
 MetricFlow provides a set of abstractions that help you construct complicated logic and dynamically generate queries to handle:
 
-- Complex metric types such as ratio, expression, and cumulative
 - Multi-hop joins between fact and dimension sources
+- Complex metric types such as ratio, expression, and cumulative
 - Metric aggregation to different time granularities
 - And so much more
 
-As a developer, you can also use MetricFlow's interfaces to construct APIs for integrations to bring metrics into downstream tools in your data stack.
-
-MetricFlow itself acts as a semantic layer, compiling the semantic information described in the MetricFlow spec to SQL that can be executed against the data warehouse and served to downstream applications. It acts as a proxy, translating metric requests in the form of “metrics by dimensions” into SQL queries that traverse the data warehouse and the underlying semantic structure to resolve every possible combination of metric and dimension.
+As a developer, you can also use MetricFlow's interfaces to construct APIs for integrations to bring metrics into downstream tools in your data stack.Note: You may need to install postgresql or graphviz. You can do so using brew: `brew install postgresql` or `brew install graphviz`
 
 ## Getting Started
 
 ### Install MetricFlow
 
-If you do not have postgres on your machine, first install Postgres:
-- Postgres provides [pre-built packages for download and installation](https://www.postgresql.org/download/)
-- Mac users might prefer to use Homebrew: `brew install postgresql`
-
-If you would like to visualize metric dataflow plans via CLI, install Graphviz:
-- Graphviz provides [pre-built packages for download and installation](https://www.graphviz.org/download/)
-- Mac users might prefer to use Homebrew: `brew install graphviz`
-
-The visualizations are in an early state of development, but look similar to:
-
-<p align="center">
-<img src="https://github.com/transform-data/metricflow/raw/main/assets/example_plan.svg" height="500"/>
-<br /><br />
-</p>
-
-Then, proceed with the regular installation as follows:
-
 MetricFlow can be installed from PyPi for use as a Python library with the following command:
 
 ```
 pip install metricflow
 ```
 
+Note: You may need to install postgresql or graphviz. You can do so using brew: `brew install postgresql` or `brew install graphviz`
```

Comment:
Let us please continue to link to the Postgres and Graphviz installation instructions, instead of restricting our instructions to homebrew users. 

Homebrew is the dominant package manager for Mac users, but it is by no means universal. It is also much less common among Windows and Linux users (and in fact was not usable on those systems until fairly recently).

Reply:
Added links to the instruction pages